gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/CredentialsController.groovy

```diff
@@ -41,26 +42,42 @@ class CredentialsController {
   @Autowired
   CredentialsService credentialsService
 
+  @Autowired
+  AccountLookupService accountLookupService
+
   @Autowired
   ObjectMapper objectMapper
 
+  static class AccountWithAuthorization extends Account {
+    Boolean authorized
+  }
+
   @PreAuthorize("@fiatPermissionEvaluator.storeWholePermission()")
   @PostFilter("hasPermission(filterObject.name, 'ACCOUNT', 'READ')")
```

Comment:
Do we need to remove this postfilter?

Reply:
This stuff only kicks in with fiat enabled (in which case the resources in those accounts also won't show up so the UI side issue should go away)